README.md

```diff
@@ -1,79 +1,285 @@
+* The name length is at least 6 and at most 50 characters.
+* The name can contain only letters, numbers, and hyphens “-“.
+* The name must start with a letter and end with a letter or number.
+* The name doesn't end with “-sb“ or “-mgmt“.
+
+4. For **Location**, choose the region in which your namespace should be hosted.
+
+5. For **Pricing tier**, select the pricing tier (Basic, Standard, or Premium) for the namespace. For this quickstart, select Standard.
+
+> **Notice:** If you want to use topics and subscriptions, choose either Standard or Premium. Topics/subscriptions aren't supported in the Basic pricing tier. If you selected the Premium pricing tier, specify the number of messaging units. The premium tier provides resource isolation at the CPU and memory level so that each workload runs in isolation. This resource container is called a messaging unit. A premium namespace has at least one messaging unit. You can select 1, 2, 4, 8, or 16 messaging units for each Service Bus Premium namespace. For more information, see [Service Bus Premium Messaging](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-premium-messaging).`
+
+6. Select **Review + create** at the bottom of the page.
+
+![Create Namespace](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-azure-service-bus/main/ballerina/resources/create-namespace.png)
+
+7. On the **Review + create** page, review settings, and select **Create**.
 
-Execute the commands below to build from the source.
 
-1. To Build & Package the Native Java Wrapper:
-   Change the current directory to the root directory and execute the following command.
+### Obtain tokens for authentication
 
-```shell script
-    ./gradlew build
+To send and receive messages from a Service Bus queue or topic, clients must use a token that is signed by a shared access key, which is part of a shared access policy. A shared access policy defines a set of permissions that can be assigned to one or more Service Bus entities (queues, topics, event hubs, or relays). A shared access policy can be assigned to more than one entity, and a single entity can have more than one shared access policy assigned to it.
+
+To obtain a token following steps should be followed:
+
+1. In the left navigation pane of the portal, select *All services*, select *Integration* from the list of categories, hover the mouse over *Service Bus*, and then select your namespace.
+
+2. In the left navigation pane of the namespace page, select *Shared access policies*.
+
+3. Click on the *RootManageSharedAccessKey* policy.
+
+4. Copy the *Primary Connection String* value and save it in a secure location. This is the connection string that you use to authenticate with the Service Bus service.
+
+![Connection String](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-azure-service-bus/main/ballerina/resources/connection-string.png)
+
+## ASB Admin Client
```

Comment:
It seems like the structure we follow in the code samples are bit different from the other connectors. Shall we update this according to the standard structure we are following for other connectors?

Reply:
Yes. This is not the convention we followed with any other connectors. 

After previous reviews (https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/206#discussion_r1402906520), I came up with this hybrid version since this is a Messaging Bus. 

But now, since the new IBM MQ also follows the same conventions as other connectors, 
we can migrate this also: +1 for that.

Changed with [ff89865](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/206/commits/ff89865bcc09cd1e41a02929615f7d6cff1ffdca).